bears/general/InvalidLinkBear.py

```diff
@@ -79,14 +81,15 @@ def find_links_in_file(file, timeout, link_ignore_regex):
             match = regex.search(line)
             if match:
                 link = match.group()
-                if not link_ignore_regex.search(link):
+                if not (link_ignore_regex.search(link) or \
```

Comment:
thanks for pointing it out... I modified it... still struggling with appveyor though...